massa-async-pool/src/test_exports/bootstrap.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use massa_models::{Address, Slot};
+
+/// This file defines tools to test the async pool bootstrap
+
+/// creates an async pool bootstrap state from components
+pub fn make_bootstrap_state(messages: Vec<AsyncMessage>) -> AsyncPoolBootstrap {
+    AsyncPoolBootstrap { messages }
+}
+
+/// asserts that two instances of AsyncMessage are the same
+pub fn assert_eq_async_message(v1: &AsyncMessage, v2: &AsyncMessage) {
```

Comment:
Should we directly implement Eq for AsyncMessage?

Reply:
We should keep it separated imo, there is no use (yet?) for being able to match the content of two messages (apart from testing).

Reply:
Exactly. I believe that testing concerns should be kept outside of the main code

Reply:
But you could hide the trait implementation behind a flag, and that would allow to use directly the == operator only in tests

Reply:
That's an option, but is the effort and added complexity really worth it given that this is used only once in tests, especially since it would not change anything except giving less information when a mismatch happens (currently the panic says which sub-field mismatched) ?

Reply:
Done.

Reply:
I disagree with the changes.

Reply:
Ok I can revert